model/fn-execution/build.gradle

```diff
@@ -17,17 +17,16 @@
  */
 
 plugins { id 'org.apache.beam.module' }
-applyPortabilityNature(
+/*applyPortabilityNature(
     automaticModuleName: 'org.apache.beam.model.fn.execution',
     shadowJarValidationExcludes: ["org/apache/beam/model/fnexecution/v1/**"]
+)*/
+
+applyJavaNature(
+        enableStrictDependencies: true,
```

Comment:
Since applyJavaNature was not declared here before, i had to explicitly create it. I copied the contents of applyPortabilityNature to applyJavaNature and too set  enableStrictDependencies: true.
@ibzib  can you please validate if it is valid approach.

Reply:
`applyJavaNature` is used for Java projects. However, `model/*` does not contain any Java source code; it only generates Java source code using [protocol buffers](https://developers.google.com/protocol-buffers). So we don't need to enable strict dependency checking on these projects.